### PR TITLE
MNT Test additional modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/660

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -43,7 +43,11 @@
             <directory>vendor/silverstripe/asset-admin/tests/php/</directory>
             <directory>vendor/silverstripe/campaign-admin/tests/php/</directory>
             <directory>vendor/silverstripe/cms/tests/</directory>
+            <directory>vendor/silverstripe/config/tests/</directory>
```
Comment (ChloeSartorelli) [inline phpunit.xml.dist:46]: config seems like core to me

File: phpunit.xml.dist [hunk 2]
```diff
@@ -43,7 +43,11 @@
...
+            <directory>vendor/silverstripe/errorpage/tests/</directory>
             <directory>vendor/silverstripe/graphql/tests/</directory>
+            <directory>vendor/silverstripe/login-forms/tests/php/</directory>
```
Comment (ChloeSartorelli) [inline phpunit.xml.dist:49]: errorpage and login-forms aren't admin - they aren't in the /admin space at all. They should go in a new testsuite perhaps.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by emteknetnz** [comment]
The testsuite names are completely arbitrary and in practice they'll never be run separately.  Their actual purpose is to facilitate parallel testing in CI.  gha-generate matrix will create jobs based on `<testsuite>` nodes

The "admin" testsuite takes the least amount of time to run right now, so it makes sense to add the new directories there

I don't want to add a new testsuite because we there's already 14 jobs being created due to the 3 PHP versions.  Also there's little point since framework alone takes 18 minutes https://github.com/silverstripe/silverstripe-installer/actions/runs/4010055028/jobs/6886120318 so the overall action won't be any faster

**Comment by GuySartorelli** [comment]
> The testsuite names are completely arbitrary

They're not, though? Like... it's fairly obvious that "admin" is "this stuff is in /admin".

But whatever, I'll just merge it.

**Pull request merged**